### PR TITLE
Update in the document

### PR DESCRIPTION
**Pull request by joinimran**
As intune is now the Endpoint protection manager, so updated the content.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10173

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - Please copy review.

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 1]
```diff
@@ -50,11 +50,11 @@ For this policy to work, you must verify that the MDM service provider allows th
 To ensure that the auto-enrollment feature is working as expected, you must verify that various requirements and settings are configured correctly.
 The following steps demonstrate required settings using the Intune service:
 
-1. Verify that the user who is going to enroll the device has a valid Intune license.
+1. Verify that the user who is going to enroll the device has a valid Endpoint Portection Manager license.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md:53]:
```suggestion
1. Verify that the user who is going to enroll the device has a valid Endpoint Protection Manager license.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
Thanks, @JohanFreelancer9 - Ready to live after copy review.

@e- FYI Please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**